components/log-viewer-webui/client/public/index.html

```diff
@@ -5,6 +5,10 @@
     <meta charset="utf-8"/>
     <meta name="description" content="YScope Log Viewer">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1">
+    <link rel="preconnect" href="https://fonts.googleapis.com"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin/>
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"/>

```

Comment:
```suggestion
          href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
      />
```

I know it's still over 100 chars, but it's a little better.